views/index.hbs

```diff
@@ -0,0 +1,8 @@
+<div class"homepage">
+  <form class="search" action="/artist-search" method="GET">
+    <label>
+      <input type ="text" name="artist" size="50"/>
+    </label>
+    <button class="submit-btn" href="http://127.0.0.1:3000/artist-search">Search for an artist</button>
```

Comment:
Never use local links in your code , if you put this code online, this localhost link does not exists there.

In this case it is enough to write href="/artist-search"